core/controllers/android_e2e_config_test.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for the admin page."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+
+from constants import constants
+from core.domain import exp_fetchers
+from core.domain import skill_fetchers
+from core.domain import story_fetchers
+from core.domain import topic_fetchers
+from core.domain import topic_services
+from core.platform import models
+from core.tests import test_utils
+import feconf
+
+(
+    audit_models, exp_models, opportunity_models,
+    user_models
+) = models.Registry.import_models([
+    models.NAMES.audit, models.NAMES.exploration, models.NAMES.opportunity,
+    models.NAMES.user
+])
+
+EXPLORATION_ID = '26'
```

Comment:
Why is this exploration_id defined as a module constant?

Reply:
removed, removed added it to a test (`exp_id`)